WIP: Rédaction du README

jnsebgosselin opened the pull request:
L'objectif de cette PR est d'ajouter une petite description du groupe et de sa vocation.

De plus, on propose d'ajouter tout de suite un code de conduite parce que ça fait sérieux et que c'est tendance :)

Pour voir le rendu final du README, aller à https://github.com/cgq-qgc/Hello/tree/hello_readme

jnsebgosselin committed:
Premier jet

jnsebgosselin committed:
Modification du titre et coquille

jnsebgosselin committed:
Correction instructions pour se joindre

FHuchet commented:
Le texte est concis et clair. On pourra le compléter par la suite au besoin.

Peut-être ajouter ou avoir un courriel "RNCAN" dans les contacts au lieu des deux adresses "INRS" ? Mais ce n'est pas bien grave...

jnsebgosselin committed:
Ajout code de conduite

FHuchet commented:
Il manque juste un "espace" avant les ":" et le "?".

jnsebgosselin committed:
Ajout d'info dans le readme

jnsebgosselin committed:
Enlever une ligne inutile

jnsebgosselin committed:
Remplacer projet ou communauté par groupe

jnsebgosselin committed:
Corriger typo et ajout section responsables

jnsebgosselin committed:
Changer CGQ pour CGQ-QGC et ajout hyperlien

jnsebgosselin committed:
Ajout liens intéressants et comment joindre

jnsebgosselin committed:
Ajout lien 2FA

jnsebgosselin committed:
Ajout lien signing commits

jnsebgosselin committed:
Ajout de plus de liens!

jnsebgosselin committed:
Réorganisation des liens en sous-groupes

jnsebgosselin committed:
Merge sec responsables avec comment se joindre

jnsebgosselin commented:
@FHuchet et @haroldvigneault 
J'ai pas mal fini ce que je voulais faire pour l'instant. Je propose de merger le tout directement dans la branche `Master` une fois que vous aurez terminer d'apporter vos corrections.

Ensuite, on devrait faire valider le tout par Pascale de la CGC et Jean-Daniel de l'INRS et on apportera leur modifications dans une autre PR.

Qu'est-ce que vous en pensez?

haroldvigneault committed:
BOn ja ai ajouter du text et j apprends

jnsebgosselin commented:
> Peut-être ajouter ou avoir un courriel "RNCAN" dans les contacts au lieu des deux adresses "INRS" ? Mais ce n'est pas bien grave..

@haroldvigneault est-ce que tu pourrais mettre ton adresse NRCAN au lieu de INRS? Je trouve que c'est une bonne idée.

jnsebgosselin commented:
> C'est super le code de conduite! L’INRS et/ou la CGC ont des codes de conduite je me demande si nous y sommes assujetti.

Est-ce que l'on devrais utiliser un de ces codes à la place? Est-ce qu'ils sont adaptés à un groupe comme celui-ci?

jnsebgosselin commented:
J'ai repensé à ca, pour éviter des problèmes, il faudrait montrer ces propositions de changements à Jean-Daniel, Pascal et J-F avant de merger.

FHuchet committed:
Update README.md

Corrections mineures apportées.

FHuchet committed:
Update README.md

FHuchet committed:
Update code-of-conduct.md

J'ai appliqué quelques modifications dont une homogénéisation des listes avec la forme "verbale".

FHuchet commented:
J'ai fait quelques modifications ce matin dans les deux documents (surtout sur la forme). Le fond me paraît vraiment bien.

jnsebgosselin reviewed (commented):
Désolé, ceci était un test pour voir si je pouvais approuver les changements, mais comme je suis l'auteur du PR, je ne peux pas évidemment...

jnsebgosselin commented:
@FHuchet je trouve que tes modifications sur le code de conduite sont très pertinentes... je me demande si ça ne vaudrait pas la peine de proposer cela "upstream"... Je peux t'aider à le faire si jamais ça t'intéresse.

jnsebgosselin committed:
Changer un lien vers le GUI pour git

FHuchet reviewed (changes requested):
Je me permets de tester le processus de révision sur GitHub.

Il y a juste l'adresse d'Harold à l'INRS, en a-t-il une à la CGC ?

jnsebgosselin commented:
>Je me permets de tester le processus de révision sur GitHub.

> Il y a juste l'adresse d'Harold à l'INRS, en a-t-il une à la CGC ?

@haroldvigneault me disait que ce ne serait pas pratique de mettre son adresse de la CGC, parce qu'il n'a que de courts contrats et qu'il ne s'en sert pas beaucoup.

jnsebgosselin committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into hello_readme

jnsebgosselin committed:
Merge branch 'hello_readme' of https://github.com/cgq-qgc/Hello into hello_readme

jnsebgosselin commented:
![image](https://user-images.githubusercontent.com/10170372/48961189-931e9300-ef40-11e8-8a9c-1f5125c87917.png)

jnsebgosselin committed:
Incorporer correction de Pascale Côté

FHuchet commented:
Merci pour les corrections de Pascale Côté.

jnsebgosselin committed:
Corriger coquille

jnsebgosselin committed:
Ajout adresse courriel du CGQ
